ocs_ci/ocs/uninstall.py

```diff
@@ -54,6 +55,55 @@ def remove_ocp_registry_from_ocs(platform):
         log.info("platform registry not supported")
 
 
+def uninstall_lso():
+    """
+    Function uninstalls local-volume objects from OCS cluster
+
+    """
+    ocp_obj = ocp.OCP()
+    storage_cluster = ocp.OCP(kind=constants.STORAGECLUSTER, resource_name=constants.DEFAULT_CLUSTERNAME)
+
+    sc_name = storage_cluster.get('spec').get('storageDeviceSets') \
+        .get().get('dataPVCTemplate').get('spec').get('storageClassName')
+    sc_obj = ocp.OCP(kind=constants.STORAGECLASS, resource_name=sc_name)
+
+    lv_name = sc_obj.get('metadata').get('labels').get('local.storage.openshift.io/owner-name')
+    lv_obj = ocp.OCP(kind=constants.LOCAL_VOLUME, resource_name=lv_name)
+    log.info(f"storage class: {sc_name}  local volume:{lv_name}")
+    device_list = lv_obj.get('spec').get('storageClassDevices').get().get('devicePaths')
+    storage_node_list = get_labeled_nodes(constants.OPERATOR_NODE_LABEL)
+
+    log.info("deleting local volume")
+    lv_obj.delete()
+
+    pv_obj_list = ocp.OCP(kind=constants.PV, namespace=constants.LOCAL_STORAGE_NAMESPACE,
+                          selector=f"storage.openshift.com/local-volume-owner-name={sc_name}")
+    log.info("deleting local volume PVs")
```

Comment:
please log the local volume details (name/path)

Reply:
done in every iteration